packages/ui/feature-dashboard/src/lib/components/alerts-table/alerts-table.component.html

```diff
@@ -5,11 +5,46 @@
         featureKey="ALERTS" videoUrl="https://cdn.activepieces.com/videos/showcase/alerts.mp4"></ap-upgrade-note>
     } @else {
     <div class="ap-h-full">
-        <div class="ap-p-[0.875rem]">
-            <span class="ap-typography-subtitle-2">Receive email alerts for issues in flow runs, quota alerts, and
-                manage the recipient emails.</span>
+        <div class="ap-w-full ap-pt-[0.375rem]">
+            <span class="ap-typography-subtitle-2">Get periodic alerts for flow failures and manage recipient emails for
+                alert notifications.</span>
         </div>
-        <ap-markdown class="ap-mb-2" [data]="betaNote"> </ap-markdown>
+        <div class="ap-px-[0.875rem] ap-py-[0.675rem] ap-my-2 ap-border ap-border-solid ap-border-gray-200">
+            <div>
+                <h3 class="ap-typography-subtitle-1">Alert Types</h3>
+                <ul class="ap-typography-body-2 ap-ml-4 ap-list-disc">
+                    <li class="ap-my-2">Immediate flow failure alerts up to 5 per hour 15 per day per flow</li>
+                    <li class="ap-mb-2">Usage limit alerts <strong>(enterprise only)</strong></li>
+                    <!-- TODO: Add when reminders is finished -->
+                    <!-- <li class="ap-mb-2">Reminders for ongoing issues <strong>(enterprise only)</strong></li> -->
+                </ul>
+            </div>
+        </div>
+
+        <div class="ap-mt-4 ap-w-full ap-flex ap-flex-col">
+            <span class="ap-typography-subtitle-1">Settings</span>
+            <mat-form-field class="ap-mt-2 ap-w-[20%]">
+                <mat-label>Alerts on Runs Frequency</mat-label>
+                <mat-select i18n-placeholder [formControl]="notificationControl">
+                    <mat-select-trigger>
+                        {{selectTriggerValue}}
```

Comment:
Instead of this why not just use a map, where the key is the possible notification value, and the value $localize`DISPLAY_NAME`